src/test/java/LoginTests.java

```diff
@@ -1,50 +1,16 @@
-import org.openqa.selenium.By;
-import org.openqa.selenium.WebElement;
-import org.testng.Assert;
 import org.testng.annotations.Test;
 import pages.HomePage;
 import pages.LoginPage;
 
 public class LoginTests extends BaseTest {
-
-    //    @Test(enabled = true, priority = 0, description = "LoginEmptyEmailPasswordTest")
-    @Test(enabled = false, dataProvider = "incorrectLoginProviders", dataProviderClass = BaseTest.class)
-    public void loginEmptyEmailPasswordTest (String email, String password) {
-        LoginPage loginPage = new LoginPage(getDriver());
-
-        loginPage.provideEmail("demo@class.com");
-        loginPage.providePassword("te$t$tudent");
-        loginPage.clickSubmitBtn();
-        Assert.assertEquals(getDriver().getCurrentUrl(), url);
-    }
-
-    //Page Object Model example
-    @Test(enabled = true, priority = 1, description = "Login with Valid Email and Password Test")
-    public void LoginValidEmailPasswordTest () {
-
-        LoginPage loginPage = new LoginPage(getDriver());
-        HomePage homePage = new HomePage(getDriver());
-
-        loginPage.provideEmail("demo@class.com")
-                .providePassword("te$t$tudent")
-                .clickSubmitBtn();
-
-        Assert.assertTrue(homePage.getUserAvatar().isDisplayed());
-
-    }
-
-    @Test(enabled = true, priority = 2, description = "Login with Invalid Email and Valid Password Test")
-    public void loginInvalidEmailValidPasswordTest () {
-        LoginPage loginPage = new LoginPage(getDriver());
-
-        loginPage.provideEmail("invalid@class.com");
-        loginPage.providePassword("te$t$tudent");
-        loginPage.clickSubmitBtn();
-
-        Assert.assertEquals(getDriver().getCurrentUrl(), url);
-
-    }
-
-
-
-}
+    @Test
+    public void LoginPageValidEmailPasswordTest() throws InterruptedException {
```

Comment:
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

For example `public void loginEmptyEmailPasswordTest ()`

and 

Please, **delete** `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.